manifest.json

```diff
@@ -1,4 +1,14 @@
 {
+  "cardanoconnect": {
+    "image": "ghcr.io/hyperledger/firefly-cardanoconnect",
+    "tag": "v0.4.1",
+    "sha": "78b1008bd62892f6eda197b5047d94e61621d0f06b299422ff8ed9b34ee5ce50"
+  },
+  "cardanosigner": {
+    "image": "ghcr.io/hyperledger/firefly-cardanosigner",
+    "tag": "v0.4.1",
+    "sha": "d0b76613ccc70ff63e68b137766eb009d589489631cee6aabf2b45e33a1ca5d3"
+  },
```

Comment:
This might need updating again

Reply:
These are the latest published versions, I just confirmed that with `manifestgen.sh`. We could probably cut new versions of those soon, but I'd rather not block this big long-lived PR on it.